identifiers/country-es.csv

```diff
@@ -53,6 +60,9 @@ ocd-division/country:es/autonomous_community:ga/province:or,Ourense
 ocd-division/country:es/autonomous_community:ga/province:po,Pontevedra
 ocd-division/country:es/autonomous_community:ib,Illes Balears
 ocd-division/country:es/autonomous_community:ib/province:pm,Baleares
+ocd-division/country:es/autonomous_community:ib/province:pm/ed:1,Mallorca
+ocd-division/country:es/autonomous_community:ib/province:pm/ed:2,Menorca
+ocd-division/country:es/autonomous_community:ib/province:pm/ed:3,Ibiza-Formentera

```

Comment:
I can't find this in the PDF. I see "Eivissa-Formentera"

Reply:
"Eivissa" is the "Catalan" name for Ibiza (spanish). 
So i would keep Ibiza in this case.